launcher/internal/gpu/deviceinfo/deviceinfo.go

```diff
@@ -0,0 +1,99 @@
+// Package deviceinfo provides functionality to retrieve information about the GPU
+// attached to the host machine. It supports identifying GPU types and determining
+// whether they have support for open-source drivers.
+package deviceinfo
```

Comment:
Why not just use cos.googlesource.com/cos/tools.git/src/cmd/cos_gpu_installer/deviceinfo? You don't have to duplicate the same logics here.

Reply:
Done.